packages/browser-embedded-sdk/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "dist/esm",
-    "moduleResolution": "bundler"
+    "moduleResolution": "bundler",
+    "isolatedModules": true
```

Comment:
Did we need isolatedModules here? Why are these settings added if this is only CI config?